### DISCUSSION TITLE
Can video subtitles be changed in language?

### DISCUSSION THREAD

**Discussion by three3in** [pystardust/ani-cli#1382] [Q&A]
I want to change the English subtitles to Chinese

**Comment by port19x** [comment]
No, that would require a different source

**Reply by natsutamura** [reply]
Is this subtitle embed into the video or is it a separate .srt file? I want to change it to Indonesian subtitles

**Reply by port19x** [reply]
embedded sadly

**Reply by candrapersada** [reply]
`Luf-Mp4` https://w2r.biananset.net from https://allmanga.to/
![image](https://github.com/user-attachments/assets/9a9b9ddb-3bf1-4346-afb0-7c95e18166d1)
subtitle `.vtt`
